eslint.config.mjs

```diff
@@ -207,7 +212,12 @@ export default [
         tsconfigRootDir: DIR_NAME,
         ecmaVersion: 'latest',
         extraFileExtensions: ['.vue'],
-        projectService: true,
+        projectService: {
+          allowDefaultProject: [
+            'app/ydoc-server/vitest.config.ts',
+            'app/ydoc-shared/vitest.config.ts',
+          ],
```

Comment:
Why do we need this here, and not in other packages?

Reply:
i have no clue honestly, maybe `vitest.config.ts` is ignored by eslint in other packages? it's definitely quite odd

Reply:
> i have no clue honestly, maybe vitest.config.ts is ignored by eslint in other packages? it's definitely quite odd

Actually, in other packages the vitest.config.ts are just listed in their tsconfigs.jsons. 

So instead here, these two file should be put in _their_ tsconfigs (`app/ydoc-server/tsconfig.json` and `app/ydoc-shared/tsconfig.json` respectively).

Reply:
that's what *i* would do too, but they are outside the `rootDir` (`src`) so `tsconfig.json` complains when i do that